### DISCUSSION TITLE
Is “Espeak-ng” License contaminating?

### DISCUSSION THREAD

**Discussion by Lurcy38** [espeak-ng/espeak-ng#1868] [General]
Hello, 
Is “Espeak-ng” License contaminating? the component seems to be release under 2 licences, GPL v3 and Apache2.
My understanding, GPL is not compatible with commercial use. Apache allow it. The commit mentions that:
**“Add the Apache 2.0 license (for the APK code in 'android').”**
What does it mean? is It  only for android project or for all developments using this module. 
I am not comfortable with these concepts, and there are no threads clarifying if I can use freely this techno in my application.
Any comments are welcome :)

**Comment by thewh1teagle** [comment]
@Lurcy38 

Did you found answer?
The current espeak-ng license is deal breaker for many people and they simply can't use it. LGPL will be easier ig.

**Comment by Please-just-dont** [comment]
I'm also wondering about this.
